helm/kubernetes-nginx-ingress-controller-chart/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: v0.15.0
+appVersion: v0.21.0
 description: A Helm chart for the nginx ingress-controller
 name: kubernetes-nginx-ingress-controller-chart
 version: 0.3.4-[[ .SHA ]]
```

Comment:
```suggestion
version: 0.4.0-[[ .SHA ]]
```

Reply:
For the new IC version a new minor version is required, otherwise it will get rolled automatically.

Reply:
good point